spec/requests/balance_pages_spec.rb

```diff
@@ -234,6 +234,16 @@
         expect(page).not_to have_text("Expected deposit on")
       end
 
+      it "shows failed payments for creator-owned Stripe Connect payouts" do
+        create(:merchant_account_stripe_connect, user: seller)
+        create(:payment_failed, processor: "STRIPE", failure_reason: "account_closed", user: seller)
+
+        visit balance_path
+
+        expect(page).to have_text("Failed")
+        expect(page).to have_text("This payout failed due to the bank account has been closed. It will be included in your next payout automatically once verification is complete.")
```

Comment:
expect(page).to have_text("This payout failed due to the bank account being closed. The amount will be included in your next payout automatically once verification is complete.")

Reply:
The current language is not grammatically correct.

Reply:
The language still seems incorrect and doesn't match the suggestion.